Taskfile.yml

```diff
@@ -187,11 +199,13 @@ tasks:
       # These files are auto-generated and ignored by the formatter
       - exclude: R/000-wrappers.R
       - exclude: R/import-standalone-*.R
+      - exclude: R/generated-*.R
```

Comment:
IIUC this is to avoid invalidating the task, but shouldn't this also be added in `air.toml`?

Reply:
It is assumed that Air ignores these files because of the `# Generated by` comment on the first line.